heft-plugins/heft-serverless-stack-plugin/src/ServerlessStackPlugin.ts

```diff
@@ -5,142 +5,152 @@ import * as path from 'path';
-      parameterLongName: '--sst',
-      // Once https://github.com/serverless-stack/serverless-stack/issues/1537 is fixed, we may be
-      // eliminate the need for this parameter.
-      description: 'Invokes the SST postprocessing - requires AWS credentials'
-    });
+    this._logger = taskSession.logger;
+
+    // Once https://github.com/serverless-stack/serverless-stack/issues/1537 is fixed, we may be
+    // eliminate the need for this parameter.
+    const sstParameter: CommandLineFlagParameter = taskSession.parametersByLongName.get(
+      '--sst'
+    ) as CommandLineFlagParameter;
+
+    const sstStageParameter: CommandLineStringParameter = taskSession.parametersByLongName.get(
+      '--sst-stage'
+    ) as CommandLineStringParameter;
+
+    // Only tap if the --sst flag is set.
+    if (sstParameter.value) {
+      taskSession.requestAccessToPluginByName(
+        '@rushstack/heft-webpack4-plugin',
+        Webpack4PluginName,
+        async (accessor: IWebpack4PluginAccessor) => {
+          accessor.onConfigureWebpackHook?.tapPromise(
+            { name: PLUGIN_NAME, stage: Number.MAX_SAFE_INTEGER },
+            async (config: IWebpack4Configuration | null) => {
+              // Discard Webpack's configuration to prevent Webpack from running
+              this._logger.terminal.writeVerboseLine(
+                'The command line includes "--sst", redirecting Webpack to Serverless Stack'
+              );
+              return null;
+            }
+          );
+        }
+      );
 
-    const sstStageParameter: IHeftStringParameter = heftSession.commandLine.registerStringParameter({
-      associatedActionNames: ['test', 'build', 'start'],
-      parameterLongName: '--sst-stage',
-      argumentName: 'STAGE_NAME',
-      description:
-        'Specifies the Serverless Stack stage; equivalent to to the "--stage" parameter from the "sst" CLI'
-    });
+      taskSession.requestAccessToPluginByName(
+        '@rushstack/heft-webpack5-plugin',
+        Webpack5PluginName,
+        async (accessor: IWebpack5PluginAccessor) => {
+          accessor.onConfigureWebpackHook?.tapPromise(
+            { name: PLUGIN_NAME, stage: Number.MAX_SAFE_INTEGER },
+            async (config: IWebpack5Configuration | null) => {
+              // Discard Webpack's configuration to prevent Webpack from running
+              this._logger.terminal.writeVerboseLine(
+                'The command line includes "--sst", redirecting Webpack to Serverless Stack'
+              );
+              return null;
+            }
+          );
+        }
+      );
 
-    let sstCliPackagePath: string;
+      taskSession.hooks.run.tapPromise(PLUGIN_NAME, async (runOptions: IHeftTaskRunHookOptions) => {
+        // TODO: Handle watch / serve mode
+        await this._runServerlessStackAsync({
+          taskSession,
+          heftConfiguration,
+          sstStage: sstStageParameter.value,
+          debugMode: taskSession.debugMode,
+          // serveMode:
```

Comment:
?

Reply:
Was done as a reminder to implement watch/serve mode. But I'll remove to keep it simple (leaving in the TODO)